### ISSUE TITLE
Error: You provided an invalid zone when listing nft with english auction

### ISSUE THREAD

**Issue by lehongvo** [ProjectOpenSea/opensea-js#1220]
### Component

API, Other (please describe)

### Have you ensured that all of these are up to date?

- [X] opensea-js
- [X] Node (minimum v16)

### What version of opensea-js are you on?

6.1.8

### What function is the bug in?

createSellOrder

### Operating System

macOS (Apple Silicon)

### Describe the bug

Please check in link
https://github.com/ProjectOpenSea/opensea-js/discussions/121

- body: '{"errors":["You provided an invalid zone: 0x110b2b128a9ed1be5ef3232d8e4e41640df5c2cd, please use the null address: 0x0000000000000000000000000000000000000000 for order placement"]}
- englishAuction: true and paymentTokenAddress: "0xa6fa4fb5f76172d178d61b04b0ecd319c5d1c0aa",
- Network Mumbai

Source code: 
```
order = await openseaSDK.createSellOrder({
                asset: {
                    tokenAddress: tokenAddress,
                    tokenId: tokenId,
                    tokenStandard: 'ERC721'
                },
                accountAddress,
                startAmount: '0.25',
                endAmount: '0.15',
                paymentTokenAddress: "0xa6fa4fb5f76172d178d61b04b0ecd319c5d1c0aa",
                englishAuction: true,
                expirationTime: Math.round(Date.now() / 1000 + 60 * 60 * 24) // One day from now
            })
```

**Comment by ryanio** [comment]
English auctions may not be supported on testnets, can you try the same code on a mainnet like polygon and see if it works?

**Comment by lehongvo** [comment]
Thank you very much!

**Comment by lehongvo** [comment]
@ryanio When check your document you have send me before 
I see still error: 
```
node /Users/nguyenductrang/Desktop/Contract/BAP/Javis/smart-contract/test/02.openSdk.spec.js
Error: bad response (status=400, headers={"date":"Tue, 10 Oct 2023 04:17:26 GMT","content-type":"application/json","content-length":"180","connection":"close","x-trace-id":"301537983045541397579692842098652762267","vary":"Accept, origin","allow":"GET, POST, HEAD, OPTIONS","x-frame-options":"DENY","x-content-type-options":"nosniff","referrer-policy":"same-origin","cross-origin-opener-policy":"same-origin","cache-control":"no-store","cf-cache-status":"DYNAMIC","set-cookie":"__cf_bm=fXvwTm_Kr3nAAzvEFNEwQDCsrlmzKTsyXfSBrhPZFyY-1696911446-0-AQHQfs1+5IyE3cqkBXVpjy7vU35I9Wr35ilUu+x25q7kqj4yPj0O/WV+VOGchAv7fsHashqp+k8D5sHLjKxhwqQ=; path=/; expires=Tue, 10-Oct-23 04:47:26 GMT; domain=.opensea.io; HttpOnly; Secure, _cfuvid=0bKgDj93Nx5BnCEOm6Wq2VnOUG6X2ZDEKELLe7eY4NY-1696911446207-0-604800000; path=/; domain=.opensea.io; HttpOnly; Secure; SameSite=None","strict-transport-security":"max-age=15552000; includeSubDomains; preload","server":"cloudflare","cf-ray":"813c0db5ea9e509a-HKG"}, body="{\"errors\":[\"You provided an invalid zone: 0x110b2b128a9ed1be5ef3232d8e4e41640df5c2cd, please use the null address: 0x0000000000000000000000000000000000000000 for order placement\"]}", requestBody="{\"parameters\":{\"offerer\":\"0x4a6f4FFd8e7164235E5aA7Db2B8425D3E3a7a165\",\"zone\":\"0x110b2b128a9ed1be5ef3232d8e4e41640df5c2cd\",\"zoneHash\":\"0x0000000000000000000000000000000000000000000000000000000000000000\",\"startTime\":\"1696911442\",\"endTime\":\"1696997840\",\"orderType\":2,\"offer\":[{\"itemType\":2,\"token\":\"0x5afe1f633eeb4d2e625d51625943b44a7e9f1434\",\"identifierOrCriteria\":\"191\",\"startAmount\":\"1\",\"endAmount\":\"1\"}],\"consideration\":[{\"itemType\":1,\"token\":\"0x7ceB23fD6bC0adD59E62ac25578270cFf1b9f619\",\"identifierOrCriteria\":\"0\",\"startAmount\":\"243750000000000000\",\"endAmount\":\"146250000000000000\",\"recipient\":\"0x4a6f4FFd8e7164235E5aA7Db2B8425D3E3a7a165\"},{\"itemType\":1,\"token\":\"0x7ceB23fD6bC0adD59E62ac25578270cFf1b9f619\",\"identifierOrCriteria\":\"0\",\"startAmount\":\"6250000000000000\",\"endAmount\":\"3750000000000000\",\"recipient\":\"0x0000a26b00c1f0df003000390027140000faa719\"}],\"totalOriginalConsiderationItems\":2,\"salt\":\"0x0000000000000000000000000000000000000000000000000000000000000000\",\"conduitKey\":\"0x0000007b02230091a7ed01230072f7006a004d60a8d4e71d599b8104250f0000\",\"counter\":\"0\"},\"signature\":\"0x6c5ceac0041f346319e32a39a2c76c0c18dada425a2a98490ac5b2b342104d47bd293f43dd7d9392576ca345df35c86c1be005a36ddbf8d98d5e0457ec84db73\",\"protocol_address\":\"0x00000000000000ADc04C56Bf30aC9d3c0aAF14dC\"}", requestMethod="POST", url="https://testnets-api.opensea.io/v2/orders/mumbai/seaport/listings", code=SERVER_ERROR, version=web/5.7.1)
    at Logger.makeError (/Users/nguyenductrang/Desktop/Contract/BAP/Javis/smart-contract/node_modules/@ethersproject/logger/lib/index.js:238:21)
    at Logger.throwError (/Users/nguyenductrang/Desktop/Contract/BAP/Javis/smart-contract/node_modules/@ethersproject/logger/lib/index.js:247:20)
    at /Users/nguyenductrang/Desktop/Contract/BAP/Javis/smart-contract/node_modules/@ethersproject/web/lib/index.js:275:36
    at step (/Users/nguyenductrang/Desktop/Contract/BAP/Javis/smart-contract/node_modules/@ethersproject/web/lib/index.js:33:23)
    at Object.next (/Users/nguyenductrang/Desktop/Contract/BAP/Javis/smart-contract/node_modules/@ethersproject/web/lib/index.js:14:53)
    at fulfilled (/Users/nguyenductrang/Desktop/Contract/BAP/Javis/smart-contract/node_modules/@ethersproject/web/lib/index.js:5:58)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {
  reason: 'bad response',
  code: 'SERVER_ERROR',
  status: 400,
  headers: {
    date: 'Tue, 10 Oct 2023 04:17:26 GMT',
    'content-type': 'application/json',
    'content-length': '180',
    connection: 'close',
    'x-trace-id': '301537983045541397579692842098652762267',
    vary: 'Accept, origin',
    allow: 'GET, POST, HEAD, OPTIONS',
    'x-frame-options': 'DENY',
    'x-content-type-options': 'nosniff',
    'referrer-policy': 'same-origin',
    'cross-origin-opener-policy': 'same-origin',
    'cache-control': 'no-store',
    'cf-cache-status': 'DYNAMIC',
    'set-cookie': '__cf_bm=fXvwTm_Kr3nAAzvEFNEwQDCsrlmzKTsyXfSBrhPZFyY-1696911446-0-AQHQfs1+5IyE3cqkBXVpjy7vU35I9Wr35ilUu+x25q7kqj4yPj0O/WV+VOGchAv7fsHashqp+k8D5sHLjKxhwqQ=; path=/; expires=Tue, 10-Oct-23 04:47:26 GMT; domain=.opensea.io; HttpOnly; Secure, _cfuvid=0bKgDj93Nx5BnCEOm6Wq2VnOUG6X2ZDEKELLe7eY4NY-1696911446207-0-604800000; path=/; domain=.opensea.io; HttpOnly; Secure; SameSite=None',
    'strict-transport-security': 'max-age=15552000; includeSubDomains; preload',
    server: 'cloudflare',
    'cf-ray': '813c0db5ea9e509a-HKG'
  },
  body: '{"errors":["You provided an invalid zone: 0x110b2b128a9ed1be5ef3232d8e4e41640df5c2cd, please use the null address: 0x0000000000000000000000000000000000000000 for order placement"]}',
  requestBody: '{"parameters":{"offerer":"0x4a6f4FFd8e7164235E5aA7Db2B8425D3E3a7a165","zone":"0x110b2b128a9ed1be5ef3232d8e4e41640df5c2cd","zoneHash":"0x0000000000000000000000000000000000000000000000000000000000000000","startTime":"1696911442","endTime":"1696997840","orderType":2,"offer":[{"itemType":2,"token":"0x5afe1f633eeb4d2e625d51625943b44a7e9f1434","identifierOrCriteria":"191","startAmount":"1","endAmount":"1"}],"consideration":[{"itemType":1,"token":"0x7ceB23fD6bC0adD59E62ac25578270cFf1b9f619","identifierOrCriteria":"0","startAmount":"243750000000000000","endAmount":"146250000000000000","recipient":"0x4a6f4FFd8e7164235E5aA7Db2B8425D3E3a7a165"},{"itemType":1,"token":"0x7ceB23fD6bC0adD59E62ac25578270cFf1b9f619","identifierOrCriteria":"0","startAmount":"6250000000000000","endAmount":"3750000000000000","recipient":"0x0000a26b00c1f0df003000390027140000faa719"}],"totalOriginalConsiderationItems":2,"salt":"0x0000000000000000000000000000000000000000000000000000000000000000","conduitKey":"0x0000007b02230091a7ed01230072f7006a004d60a8d4e71d599b8104250f0000","counter":"0"},"signature":"0x6c5ceac0041f346319e32a39a2c76c0c18dada425a2a98490ac5b2b342104d47bd293f43dd7d9392576ca345df35c86c1be005a36ddbf8d98d5e0457ec84db73","protocol_address":"0x00000000000000ADc04C56Bf30aC9d3c0aAF14dC"}',
  requestMethod: 'POST',
  url: 'https://testnets-api.opensea.io/v2/orders/mumbai/seaport/listings'
}
```

Full source code


```
const { ethers } = require('ethers')
const { OpenSeaSDK, Chain } = require('opensea-js')
const { TokenStandard } = require('opensea-js/lib/types')
const axios = require('axios')

const createSellAction = async () => {
    try {
        const provider = new ethers.providers.JsonRpcProvider(
            'https://rpc-mumbai.maticvigil.com'
        )

        const walletWithProvider = new ethers.Wallet(
            'f6ac3a901d2170e9fa165491ca443052e6d63d50460b497aa697ef9dca194075',
            provider
        )

        const openseaSDK = new OpenSeaSDK(walletWithProvider, {
            chain: Chain.Mumbai
        })

        const accountAddress = '0x4a6f4FFd8e7164235E5aA7Db2B8425D3E3a7a165'
        const tokenId = '191'
        const tokenAddress = '0x5AFe1f633EEB4d2e625D51625943b44A7e9F1434'
        let order;
        order = await openseaSDK.createSellOrder({
            asset: {
                tokenAddress: tokenAddress,
                tokenId: tokenId,
                tokenStandard: 'ERC721'
            },
            accountAddress,
            startAmount: '0.25',
            endAmount: '0.15',
            paymentTokenAddress: "0x7ceB23fD6bC0adD59E62ac25578270cFf1b9f619",
            englishAuction: true,
            expirationTime: Math.round(Date.now() / 1000 + 60 * 60 * 24) // One day from now
        })
        console.log("order", order);
        return order
    } catch (error) {
        console.log(error)
    }
}

createSellAction();
```

Package.json
```
"devDependencies": {
    "@nomicfoundation/hardhat-chai-matchers": "^1.0.3",
    "@nomicfoundation/hardhat-toolbox": "^2.0.0",
    "@nomiclabs/hardhat-ethers": "^2.2.3",
    "@nomiclabs/hardhat-etherscan": "^3.1.0",
    "@openzeppelin/contracts": "^4.7.3",
    "@openzeppelin/contracts-upgradeable": "^4.8.0-rc.1",
    "@typechain/ethers-v5": "^10.1.0",
    "@typechain/hardhat": "^6.1.3",
    "@types/chai": "^4.3.3",
    "@types/jest": "^29.0.3",
    "chai": "^4.3.6",
    "dotenv": "^16.0.2",
    "erc721a": "^4.2.3",
    "ethers": "^5.7.2",
    "hardhat": "^2.11.2",
    "hardhat-contract-sizer": "^2.6.1",
    "hardhat-gas-reporter": "^1.0.9",
    "jest": "^29.0.3",
    "solidity-coverage": "^0.8.2",
    "ts-node": "^10.9.1",
    "typechain": "^8.1.0",
    "typescript": "^4.8.3"
  },
  "dependencies": {
    "@chainlink/contracts": "^0.6.1",
    "@opensea/stream-js": "^0.1.2",
    "@openzeppelin/hardhat-upgrades": "^1.19.0",
    "axios": "^1.5.1",
    "node-fetch": "^2.6.5",
    "opensea-js": "6.1.10",
    "ws": "^8.14.2"
  }
```

**Comment by lehongvo** [comment]
@ryanio Can you help us fix this error? Because it is urgent for us. Thank you 🙏

**Comment by ryanio** [comment]
@lehongvo you'll have to wait until we merge the PR and release the new version, or build it yourself

**Comment by lehongvo** [comment]
@ryanio when we receive new latest version?

**Comment by ryanio** [comment]
@lehongvo i'll release today and let you know when it's out

**Comment by ryanio** [comment]
This is now released in [v6.1.11](https://github.com/ProjectOpenSea/opensea-js/releases/tag/v6.1.11), please give it a try and let us know if it works for you.
